requirements.txt

```diff
@@ -1,7 +1,7 @@
 numpy==1.26.0
 sympy==1.12
 requests==2.31.0
-monty==2024.2.26
+monty>=2024.5.15
```

Comment:
Updated the `monty` version to use the `deadline` argument, wondering if there is a reason to ping a specific version instead of a minimum version?